one.ablaze.floorp.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>one.ablaze.floorp</id>
+    <metadata_license>MPL-2.0</metadata_license>
```

Comment:
For the metadata license (THIS file) `CC0-1.0` is preferred.

Reply:
and it is missing the `project_license`

Reply:
ok
I fixed it